### PR TITLE
Remove Secrets

### PR DESCRIPTION
**Pull request by ryuyu**
Reopening https://github.com/NuGet/NuGetGallery/pull/3202 against the right branch.
Address the issues that scanning for credentials on our repository found.

@skofman1 @maartenba

### CODE REVIEW

**Comment by scottbommarito** [comment]
LGTM

File: tests/NuGetGallery.Facts/Infrastructure/LuceneSearchServiceFacts.cs [hunk 1]
```diff
@@ -493,6 +493,7 @@ public void IndexAndSearchRetrievesCanDriveV2Feed()
                 DownloadCount = 12345,
                 FlattenedDependencies = "adjunct-System.FluentCast:1.0.0.4|xunit:1.8.0.1545|adjunct-XUnit.Assertions:1.0.0.5|adjunct-XUnit.Assertions.Linq2Xml:1.0.0.3",
                 HashAlgorithm = "SHA512",
+                // This is a test hash
```
Comment (scottbommarito) [inline tests/NuGetGallery.Facts/Infrastructure/LuceneSearchServiceFacts.cs]: maybe `This is a test hash and is not a secret` to be more specific?

**Comment by shishirx34** [comment]
Looks good.

File: src/NuGetGallery/Web.config [hunk 2]
```diff
@@ -52,8 +52,8 @@
     <add key="Gallery.AzureCdnHost" value="" />
     <!-- Set this to the Azure CDN Host you are using for blob storage, if you have configured one -->
     <add key="Gallery.SiteRoot" value="http://nuget.localtest.me/" />
-    <add key="reCAPTCHA::PrivateKey" value="6LcebdwSAAAAAI5dI90LpPMFGKDxdCt6fjwEwMut" />
-    <add key="reCAPTCHA::PublicKey" value="6LcebdwSAAAAAFI06sH9RRb2VP1HFCjYpg74lKG7" />
```
Comment (maartenba) [inline src/NuGetGallery/Web.config]: Store these in Key Vault?

**Comment by maartenba** [comment]
:shipit:

File: src/NuGetGallery/Web.config (hunk 2 above)
Comment (ryuyu) [inline src/NuGetGallery/Web.config]: That was the plan.

**Pull request merged**